docs/DeveloperGuide.md

```diff
@@ -1115,10 +1115,15 @@ Deleting an existing appointment from a patient
 
 Team size: 5
 
-1\. **Allow names to include symbols and special characters**
+1\. **Enhance Name Validation to Support Special Characters and Improve Formatting**
```

Comment:
remove caps from all words in "Support Special characters and Improve Formatting"

Reply:
self amend